components/application-configuration/api/src/main/resources/ApplicationResources_ja.properties

```diff
@@ -1,24 +1,24 @@
-phenotips.DBConfigurationClass.dateExample=\uff08\u4f8b\uff1a\u4eca\u65e5\u306f {0}\uff09
-PhenoTips.DBConfigurationClass_measurementUnitSystem_metric=\u30e1\u30fc\u30c8\u30eb\u6cd5
-PhenoTips.DBConfigurationClass_measurementUnitSystem_imperial=\u30a4\u30ae\u30ea\u30b9\u5f0f\u5358\u4f4d
-PhenoTips.DBConfigurationClass_phenotypeMapping=\u8868\u73fe\u578b\u69cb\u6210\u30c9\u30ad\u30e5\u30e1\u30f3\u30c8
-PhenoTips.DBConfigurationClass_dateOfBirthFormat=\u8a95\u751f\u65e5\u306e\u5f62\u5f0f
-PhenoTips.DBConfigurationClass_enableImperialUnitSystem=\u30a4\u30ae\u30ea\u30b9\u5f0f\u5358\u4f4d\u7cfb\u3067\u306e\u5024\u306e\u5165\u529b\u3092\u6709\u52b9\u306b\u3057\u307e\u3059\u3002
-PhenoTips.DBConfigurationClass_measurementUnitSystem=\u300c\u82f1\u56fd\u5f0f\u5358\u4f4d\u7cfb\u300d\u6709\u52b9\u6642\u306e\u3001\u65e2\u5b9a\u306e\u5165\u529b\u5358\u4f4d\u7cfb
-PhenoTips.DBConfigurationClass_livetableColumns=\u30e9\u30a4\u30d6\u30c6\u30fc\u30d6\u30eb\u30b3\u30e9\u30e0
-PhenoTips.DBConfigurationClass_livetableSortColumn=\u30c7\u30d5\u30a9\u30eb\u30c8\u3067\u30bd\u30fc\u30c8\u3059\u308b\u5217
-PhenoTips.DBConfigurationClass_livetableRows=\u30c7\u30d5\u30a9\u30eb\u30c8\u3067\u8868\u793a\u3059\u308b\u884c\u6570
-phenotips.DBConfigurationClass.metaInfo=\u30e1\u30bf\u60c5\u5831\uff1a
-phenotips.DBConfigurationClass.patientInfo=\u60a3\u8005\u60c5\u5831\:
+phenotips.DBConfigurationClass.dateExample=（例：今日は {0}）
+PhenoTips.DBConfigurationClass_measurementUnitSystem_metric=メートル法
+PhenoTips.DBConfigurationClass_measurementUnitSystem_imperial=イギリス式単位
+PhenoTips.DBConfigurationClass_phenotypeMapping=表現型構成ドキュメント
+PhenoTips.DBConfigurationClass_dateOfBirthFormat=誕生日の形式
+PhenoTips.DBConfigurationClass_enableImperialUnitSystem=イギリス式単位系での値の入力を有効にします。
+PhenoTips.DBConfigurationClass_measurementUnitSystem=「英国式単位系」有効時の、既定の入力単位系
+PhenoTips.DBConfigurationClass_livetableColumns=ライブテーブルコラム
+PhenoTips.DBConfigurationClass_livetableSortColumn=デフォルトでソートする列
+PhenoTips.DBConfigurationClass_livetableRows=デフォルトで表示する行数
+phenotips.DBConfigurationClass.metaInfo=メタ情報：
+phenotips.DBConfigurationClass.patientInfo=患者情報\:
```

Comment:
@sdumitriu will this look fine in PhenoTips?

Reply:
FTR this is what I see:

![image](https://user-images.githubusercontent.com/651980/56672031-aebece80-6683-11e9-9c6f-17f88752421b.png)

Reply:
It will not. Crowdin has changed in a way that is broken for us. We can either:
- manually re-escape every before merging every pull request, but they will then make another one just to unescape
- commit our own override of the XWiki class responsible for loading translations so that it can also read UTF8 files
- switch to a different l10n platform

I think the second is the best.

Reply:
+1 for option 2.